UnitTest/TestComponentCSharp_Tests.cs

```diff
@@ -297,8 +297,26 @@ public void TestVectorCastConversion()
             var uriVector = vector.Cast<Uri>();
             var first = uriVector.First();
             Assert.Equal(vector, uriVector);
-        }
-
+        }
+
+        async Task LookupPorts()
+        {
+            var ports = await Windows.Devices.Enumeration.DeviceInformation.FindAllAsync(
+                Windows.Devices.SerialCommunication.SerialDevice.GetDeviceSelector(),
+                new string[] { "System.ItemNameDisplay" });
+            foreach (var port in ports)
+            {
+                object o = port.Properties["System.ItemNameDisplay"];
+                Assert.NotNull(o);
+            }
+        }
+
+        [Fact]
+        public void TestReadOnlyDictionaryLookup()
+        {
+            Assert.True(LookupPorts().Wait(10000));

```

Comment:
any reason the timeout is so long?

Reply:
Reduced it, I think I previously needed a higher timeout as I had a slightly different logic for lookup before and had breakpoints set.  But 1000ms seems to work fine.